pkg/kubectl/cmd/create.go

```diff
@@ -71,6 +73,11 @@ func NewCmdCreate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	usage := "Filename, directory, or URL to file to use to create the resource"
 	kubectl.AddJsonFilenameFlag(cmd, &options.Filenames, usage)
 	cmd.MarkFlagRequired("filename")
+
+	usage = "URL or file with a list of resource file paths"
+	kubectl.AddJsonFilelistFlag(cmd, &options.Filesfrom, usage)
+	cmd.MarkFlagRequired("files-from")
```

Comment:
I'm not confident we should mark both flags as required. 

When you mark a flag required, [tab][tab] will only suggest the other flags available in the command when *all* required ones were provided. So it's good for when you have something that is *always* required (like `--filename` was until now). But in this new scenario we need at least one of `--filename` or `--files-from`, but none of them are actually mandatory if you provided the other. For the sake of completions I'd just not `MarkFlagRequired` in *any* of them and make sure we have a validation in place that guarantees we have at least one or another.